samples/CommunityToolkit.Maui.Sample/Platforms/Android/MainActivity.cs

```diff
@@ -1,9 +1,21 @@
 ﻿using Android.App;
 using Android.Content.PM;
+using Android.OS;
 
 namespace CommunityToolkit.Maui.Sample;
 
 [Activity(Theme = "@style/Maui.SplashTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation | ConfigChanges.UiMode | ConfigChanges.ScreenLayout | ConfigChanges.SmallestScreenSize)]
 public class MainActivity : MauiAppCompatActivity
 {
+	protected override void OnPostCreate(Bundle? savedInstanceState)
+	{
+		try
+		{
+			base.OnPostCreate(savedInstanceState);
+		}
+		catch(Exception)
+		{
```

Comment:
wondering if we should log at least the exception message... Also, friendly reminder to add this on our docs, since this should be on app project.

Reply:
Thanks @pictos for the recommendation, we added a Trace to show the error message. It has been applied. For the documentation, let me touch base with @bijington to give me some insights how to add this to the documentation.